src/main/java/com/mindee/parsing/common/ocr/OcrPage.java

```diff
@@ -0,0 +1,78 @@
+package com.mindee.parsing.common.ocr;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.mindee.geometry.PolygonUtils;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.stream.Collectors;
+import lombok.Getter;
+
+/**
+ * Represents a page.
+ */
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OcrPage {
+
+  @JsonProperty("all_words")
+  private ArrayList<Word> words = new ArrayList<>();
```

Comment:
It is generally recommended to code to the List interface, as opposed to ArrayList directly.
private List<Word> words = new ArrayList<>();